lib/rounds.js

```diff
@@ -3,8 +3,18 @@ class Round {
     this.categories = [];
     this.clues = []; 
     this.createCategories();
+    this.dailyDouble = null;
+    this.assignDailyDouble();
   };
 
+  assignDailyDouble(){
+    let randomIndex = Math.floor(Math.random() * this.clues.length);
+    let randomDailyDouble = this.clues[randomIndex]
+    this.dailyDouble = new DailyDouble(randomDailyDouble.question, randomDailyDouble.pointValue, randomDailyDouble.answer, randomDailyDouble.categoryId)
+    this.clues[randomIndex] = this.dailyDouble
+    console.log(this.dailyDouble)
+  }
```

Comment:
Good thinking on creating the random index and then using it to find that clue within the this.clues array!